packages/react-vega/src/Vega.jsx

```diff
@@ -119,16 +121,12 @@ class Vega extends React.Component {
       const { props } = this;
       // Parse the vega spec and create the view
       try {
-        const specJson = JSON.stringify(spec);
         let view = {};
         try {
-          console.log(vegaEmbed);
           result = await vegaEmbed(this.element, spec);
```

Comment:
The reason we want to use embed is that we can use the embed options. This is missing in this PR. Can you add it?

Reply:
Hmm perhaps do that in another PR -- see https://github.com/vega/react-vega/pull/38#issuecomment-495728470.

Reply:
Okay, but we should at least pass properties to embed instead of duplicating implementations.